azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -995,6 +997,21 @@ private void ajaxFetchExecutableFlow(final HttpServletRequest req,
     ret.put("execid", exFlow.getExecutionId());
     ret.put("projectId", exFlow.getProjectId());
     ret.put("project", project.getName());
+    
+    String autoRetryStatuses = exFlow.getExecutionOptions().getFlowParameters()
+        .getOrDefault(FlowParameters.FLOW_PARAM_ALLOW_RESTART_ON_STATUS, "");
+    if (!autoRetryStatuses.isEmpty()){
+      ret.put(EventReporterConstants.SYSTEM_DEFINED_FLOW_RETRY_COUNT_PARAM,
+          exFlow.getSystemDefinedRetryCount());
+      ret.put(EventReporterConstants.USER_DEFINED_FLOW_RETRY_COUNT_PARAM,
+          exFlow.getUserDefinedRetryCount());
+      ret.put(EventReporterConstants.FLOW_RETRY_ROOT_EXECUTION_ID,
+          exFlow.getFlowRetryRootExecutionID());
+      ret.put(EventReporterConstants.FLOW_RETRY_PARENT_EXECUTION_ID,
+          exFlow.getFlowRetryParentExecutionID());
+      ret.put(EventReporterConstants.FLOW_RETRY_CHILD_EXECUTION_ID,
+          exFlow.getFlowRetryChildExecutionID());
+    }
```

Comment:
Since these are multiple related fields, what do you think about grouping them together in a JSON object and perhaps also drop `flowRetry` from the properties names as it might be redundant (the api `fetchexecflow` retrieves information about the "flow"). When reporting events if wanted we could prefix the properties with "flowRetry". Here is an example api response with previous suggestions:
```
...
"retries": {
    "userDefinedCount": 1,
    "systemDefinedCount": 0,
    "rootExecutionID": 1234,
    "parentExecutionID: 1234,
    "childExecutionID: 0
}
...
```